packages/blocks/store/src/lib/actions/common.ts

```diff
@@ -37,19 +37,19 @@ export const common = {
     options: {
       options: [
         {
```

Comment:
Changed the order to match the default, it should go from most restrictive to less